examples/gcp/variables.tf

```diff
@@ -19,7 +19,7 @@ variable "region" {
 variable "orchestratord_version" {
   description = "Version of the Materialize orchestrator to install"
   type        = string
-  default     = "v0.130.4"
+  default     = "v0.130.7"
```

Comment:
We should probably be defaulting this to null, so that we use whatever version the helm chart defaults to.

Reply:
Heh ... thought I could remove in this commit ... but, reverted that since the example main.tf need ot change and such.  
A cleanup for another day.